src/components/Stories/__stories__/Stories.stories.tsx

```diff
@@ -62,6 +62,7 @@ const DefaultTemplate: StoryFn<StoriesProps> = (props: StoriesProps) => {
             <Stories
                 {...props}
                 open={visible}
+                syncInTabs={props.syncInTabs}
```

Comment:
Above, with the restructuring of props, `syncInTabs` also will be set, right?

Reply:
yes, right. Fixed.